Nemt/nemt.h

```diff
@@ -5,10 +5,14 @@
 #include <Pixy.h>
 
 #define CAMERA_CENTER 160
+#define NOT_FOUND 404
+#define DEFAULT_COUNTER_LIMIT 50
+#define CAMERA_WIDTH 320
+#define CAMERA_HORIZONTAL_ANGLE 70
```

Comment:
Also, wouldn't it be better to allow per-camera settings (the `CAMERA_*` macros) to be specified by user? That is, I think it would be better to put those as arguments for constructor.